filter.lua

```diff
@@ -117,7 +102,11 @@ local function filter(outbox)
 
 				local message = format('domain="%s",dport="%d",action="%s",reason="%s"',
 					domain, dport, verdict.action, verdict.reason)
-				outbox:send(message)
+				outbox:send('TELE' .. message)
+
+				if dport == 443 and verdict.action == "DROP" then
+					outbox:send('RPLY' .. packet:getstring(0))
```

Comment:
wondering if wouldn't be better to have a separate mailbox for this.. mixed feelings.. what do you think?